wasp/apps/flashlight.py

```diff
@@ -21,19 +20,31 @@ class TorchApp(object):
 
     def foreground(self):
         """Activate the application."""
-        self.draw()
         wasp.system.request_tick(1000)
+        wasp.system.request_event(wasp.EventMask.TOUCH)
 
         self._brightness = wasp.system.brightness
         wasp.system.brightness = 3
+        self.n_touch = 0
+        self.draw()
 
     def background(self):
-        """De-activate the application (without losing state)."""
+        """De-activate the application (without losing original state)."""
         wasp.system.brightness = self._brightness
 
     def tick(self, ticks):
         wasp.system.keep_awake()
 
     def draw(self):
         """Redraw the display from scratch."""
-        wasp.watch.drawable.fill(0xffff)
+        if self.n_touch % 6 < 3:
+            wasp.watch.drawable.fill(0xffff)
+        else:
+            wasp.watch.drawable.fill(0xf800)
```

Comment:
Ideally we should skip redrawing if the previous colour was the same (ie. only redraw at `n_touch % 3 == 0`)

Reply:
I thought it was best to make the code as small as possible instead of as power saving as possible.

Reply:
It depends! In this case I think adding that code is worth it, since redrawing takes a lot of time and in the meantime the watch will be unresponsive

Reply:
I don't know if redrawing "red" over "red" is automatically skipped but currently it's super fast on my watch.

I'll add your fix tho

Reply:
Done. Actually I ended up removing draw() entirely to make it more compact.

I also request tick every 3 seconds instead of 1.

36a267b